src/app/cli/src/commands/system_api_server_run_command.rs

```diff
@@ -155,13 +154,17 @@ impl Command for APIServerRunCommand {
                 s(format!("http://{}", api_server.local_addr())).bold(),
             );
             eprintln!("{}", s("Use Ctrl+C to stop the server").yellow());
+        }
 
-            if self.get_token {
+        if self.get_token {
+            if self.output_config.is_tty && self.output_config.verbosity_level == 0 {
                 eprintln!(
                     "{} {}",
-                    s("JWT token:").green().bold(),
+                    s("Access token:").green().bold(),
                     s(access_token).dim()
                 );
+            } else {
+                tracing::warn!("Access token: {access_token}");
```

Comment:
info?

Reply:
I want token message to be clearly visible